operator_ui/src/components/JobRuns/StatusCard.tsx

```diff
@@ -36,17 +38,56 @@ const styles = (theme: Theme) =>
     },
     statusText: {
       display: 'inline-block',
-      paddingLeft: theme.spacing.unit * 2,
-      textTransform: 'capitalize'
+      textTransform: 'capitalize',
+      color: '#3d5170'
+    },
+    statusRoot: {
+      paddingLeft: theme.spacing.unit * 2
+    },
+    elapsedText: {
+      color: '#818ea3'
+    },
+    earnedLink: {
+      color: '#17c671'
     }
   })
 
 interface IProps extends WithStyles<typeof styles> {
   title: string
   children?: React.ReactNode
+  jobRun?: IJobRun
+}
+
+const renderElapsedTime = (jobRun: IJobRun, classes) => {
+  const elapsedSecs =
+    jobRun && +new Date(jobRun.finishedAt) - +new Date(jobRun.createdAt)
+  const hours = Math.floor(elapsedSecs / 3600)
+  const minutes = Math.floor((elapsedSecs % 3600) / 60)
+  const seconds = Math.ceil((elapsedSecs % 3600) % 60)
+  return (
+    <>
+      {jobRun && jobRun.finishedAt && jobRun.createdAt && (
+        <Typography className={classes.elapsedText} variant="h6">
+          {!!hours && `${hours}h`}
```

Comment:
How do you feel about extracting a util function that can be tested in isolation for this date string formatting and then an `<ElapsedTime>` component? This will also be useful in the explorer.